src/air.rs

```diff
@@ -323,7 +323,7 @@ impl AsmLine {
         let (offset, _) = label_pos.overflowing_sub(self.line);
         let offset = (offset as i16) - 1;
         // Must fit in specified offset bits
-        if offset.abs() > 2i16.pow(bits - 1) - 1 {
+        if offset.abs() > 2i16.pow(bits - 1) - if offset > 0 { 1 } else { 0 } {
```

Comment:
This should be part of a different PR. Also, don't forget to run tests to check if everything passes after making changes.

Reply:
Thanks I didn't realize I committed this on the same branch 😂

Reply:
👍